x-pack/plugins/apm/public/services/rest/apm/services.ts

```diff
@@ -32,19 +34,39 @@ export async function loadServiceDetails({
   serviceName,
   start,
   end,
-  kuery
+  uiFilters
 }: {
   serviceName: string;
   start: string;
   end: string;
-  kuery: string | undefined;
+  uiFilters: UIFilters;
 }) {
   return callApi<ServiceAPIResponse>({
     pathname: `/api/apm/services/${serviceName}`,
     query: {
       start,
       end,
-      esFilterQuery: await getEncodedEsQuery(kuery)
+      uiFiltersES: await getUiFiltersES(uiFilters)
+    }
+  });
+}
+
+export async function loadServiceEnvironments({
+  serviceName,
+  start,
+  end
+}: {
+  serviceName?: string;
+  start: string;
+  end: string;
+}) {
+  const pathname = `/api/apm/services/environments`;
+  return callApi<ServiceEnvironmentsAPIResponse>({
+    pathname,
+    query: {
+      start,
+      end,
+      serviceName
```

Comment:
If you end up doing https://github.com/elastic/kibana/pull/35778/files#r281866312 this method should also be moved to a `filters.ts` file.